code/java/pipelines-workshop/src/main/java/com/myorg/PipelineStack.java

```diff
@@ -1,25 +1,28 @@
 package com.myorg;
 
+import java.util.Arrays;
+import java.util.Map;
+import java.util.HashMap;
+
 import software.amazon.awscdk.core.Construct;
 import software.amazon.awscdk.core.Stack;
 import software.amazon.awscdk.core.StackProps;
 
 import software.amazon.awscdk.services.codecommit.Repository;
-
 import software.amazon.awscdk.services.codepipeline.Artifact;
+import software.amazon.awscdk.services.codepipeline.actions.CodeCommitSourceAction;
 import software.amazon.awscdk.pipelines.CdkPipeline;
 import software.amazon.awscdk.pipelines.SimpleSynthAction;
 import software.amazon.awscdk.pipelines.CdkStage;
 import software.amazon.awscdk.pipelines.ShellScriptAction;
+import software.amazon.awscdk.pipelines.StackOutput;
 
-import software.amazon.awscdk.services.codepipeline.actions.CodeCommitSourceAction;
-
-public class WorkshopPipelineStack extends Stack {
```

Comment:
Java would throw an error if the class did not match the filename. To resolve this I renamed the class to PipelineStack.